### ISSUE TITLE
Failed to install chrome driver

### ISSUE THREAD

**Issue by krishna3008** [SergeyPirogov/webdriver_manager#226]
My code:
`from selenium import webdriver
from webdriver_manager.chrome import ChromeDriverManager

driver = webdriver.Chrome(ChromeDriverManager().install())
driver.close()`

Below is the Console output

====== WebDriver manager ======
Current google-chrome version is 93.0.4577
Get LATEST driver version for 93.0.4577
Traceback (most recent call last):
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\connectionpool.py", line 382, in _make_request
    self._validate_conn(conn)
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\connectionpool.py", line 1010, in _validate_conn
    conn.connect()
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\connection.py", line 416, in connect
    self.sock = ssl_wrap_socket(
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "C:\Program Files\Python39\lib\ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "C:\Program Files\Python39\lib\ssl.py", line 1040, in _create
    self.do_handshake()
  File "C:\Program Files\Python39\lib\ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\requests\adapters.py", line 439, in send
    resp = conn.urlopen(
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\urllib3\util\retry.py", line 574, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='chromedriver.storage.googleapis.com', port=443): Max retries exceeded with url: /LATEST_RELEASE_93.0.4577 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\qjq2g6\Desktop\Chrmoe_download.py", line 4, in <module>
    driver = webdriver.Chrome(ChromeDriverManager().install())
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\webdriver_manager\chrome.py", line 34, in install
    driver_path = self._get_driver_path(self.driver)
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\webdriver_manager\manager.py", line 23, in _get_driver_path
    driver_version = driver.get_version()
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\webdriver_manager\driver.py", line 43, in get_version
    return self.get_latest_release_version()
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\webdriver_manager\driver.py", line 66, in get_latest_release_version
    resp = requests.get(f"{self._latest_release_url}_{self.browser_version}")
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\requests\api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\requests\api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\requests\sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\requests\sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "C:\Users\qjq2g6\AppData\Roaming\Python\Python39\site-packages\requests\adapters.py", line 514, in send
    raise SSLError(e, request=request)
requests.exceptions.SSLError: HTTPSConnectionPool(host='chromedriver.storage.googleapis.com', port=443): Max retries exceeded with url: /LATEST_RELEASE_93.0.4577 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)')))

**Comment by aleksandr-kotlyar** [comment]
@krishna3008 are you running from proxy? Or from VPN? Or from corporate network?
It was a world-global issue 30 september - 1 october with certificates. Check this out https://www.google.com/search?q=ssl+error+letsencrypt

What version of openssl do you use? Probably check for your openssl library version - and update it - this can help if you are not behind the proxy/vpn.
https://www.openssl.org/blog/blog/2021/09/13/LetsEncryptRootCertExpire/

In CI tests are passing and from local network too. So it seems it's your local issue. Please check the articles from above and come back if you solve/not solve the issue.

**Comment by miguelius** [comment]
This can be fixed using pr #225.

 ```python
driver = webdriver.Chrome(ChromeDriverManager().dont_verify_ssl().install())
driver.close()
```

If you need to parametrize the request session, you can call the function session() and you'll get the session used by both the Manager and download_file function.

**Comment by TshokeloMokubi** [comment]
> 



> This can be fixed using pr #225.
> 
> ```python
> from webdriver_manager.chrome import ChromeDriverManager
> 
> driver = webdriver.Chrome(ChromeDriverManager().dont_verify_ssl().install())
> driver.close()
> ```
> 
> If you need to parametrize the request session, you can call the function session() and you'll get the session used by both the Manager and download_file function.

Is it safe to use 'dont_verify_ssl()'?

**Comment by miguelius** [comment]
Not really. Just under the context of a corporate proxy could be acceptable.

Another option is to tell requests to trust proxy's root-ca.

First's a compromise backed by your organization's network security. Second
is safer.
